CHANGELOG.rst

```diff
@@ -35,22 +35,29 @@ User-Facing Changes
 - **UI**: Use namespace/slug to identify Renku 2.0 projects
   (`#3103 <https://github.com/SwissDataScienceCenter/renku-ui/pull/3103>`_).
 
+
 Internal Changes
 ~~~~~~~~~~~~~~~~
 
 **🌟 New Features**
 
 - **Data Services**: Updated the authorization for access to Renku 2.0 projects to work with Authzed DB,
   a 3rd party database dedicated to saving authorization data and making authorization decisions
+- **Data Services**: Updated the authorization for access to Renku 2.0 namespaces and groups with Authzed DB
```

Comment:
Looks like something funny is going on here?

Reply:
Ooh was this intentional? It looks like a typo. Can we please combine these into one line?
Also, what do you mean by authorization to namespaces? What does that mean distinct from groups? I would expect it to say "...authorization for access to Renku 2.0 projects and groups..."